lib/relay/protocol/protocol.go

```diff
@@ -58,8 +58,6 @@ func WriteMessage(w io.Writer, message interface{}) error {
 		return err
 	}
 
-	header.messageLength = int32(len(payload))
```

Comment:
Nope, but I don't think I intended to remove it here. I guess it will have to stay in the protocol for good now